File: src/main/java/org/teasoft/exam/bee/osql/entity/Tables.java
package org.teasoft.exam.bee.osql.entity;

import java.io.Serializable;

/**
*@author Honey
*Create on 2021-10-16 23:31:57
*/
public class Tables implements Serializable {

	private static final long serialVersionUID = 1590644473308L;

	private Integer tableId;
	private String tableName;

	public Integer getTableId() {
		return tableId;
	}

	public void setTableId(Integer tableId) {
		this.tableId = tableId;
	}

	public String getTableName() {
		return tableName;
	}

	public void setTableName(String tableName) {
		this.tableName = tableName;
	}

	 public String toString(){	
		 StringBuffer str=new StringBuffer();	
		 str.append("Tables[");			
		 str.append("tableId=").append(tableId);		 
		 str.append(",tableName=").append(tableName);		 
		 str.append("]");			 
		 return str.toString();			 
	 }		 
}

File: src/main/java/org/teasoft/exam/bee/osql/special/kotlin/Tables.kt
package org.teasoft.exam.bee.osql.special.kotlin

import org.teasoft.bee.osql.annotation.JoinTable;
import org.teasoft.bee.osql.annotation.JoinType;
import java.io.Serializable;

data class Tables(
var tableId: Int ?= null,
var tableName : String ?= null,
@JoinTable (mainField = "table_id",
subField = "table_id",
joinType = JoinType.JOIN
,subClass = "Columns"
)
//var columns:List<Columns> ?= null,
var columns:Columns ?= null,
): Serializable

